docs/components/custom/MarkdownHeading/MarkdownHeading.jsx

```diff
@@ -12,7 +12,7 @@ const MarkdownHeading = ({ level, children }) => (
 
 MarkdownHeading.propTypes = {
   level: PropTypes.string.isRequired,
-  children: PropTypes.string.isRequired,
+  children: PropTypes.array.isRequired,
```

Comment:
I think the more appropriate PropTypes validator here would be `PropTypes.node`. Array doesn't actually make much sense for `children`. I think the error message React is giving is a bit misleading.